mqtt.md

```diff
@@ -1,4 +1,4 @@
-This library allows for the execution of async ClearBlade messaging functions by providing an MQTT client that will work asynchronously. You can connect to the internal ClearBlade broker or an external broker. 
+ This library allows for the execution of async ClearBlade messaging functions by providing an MQTT client that will work asynchronously. You can connect the ClearBlade internal broker to the same system where the code resides, or to an external broker (e.g. ClearBlade broker in another system, MosQuiTTo etc.).
```

Comment:
It's Mosquitto, it doesn't have to be a mixture of upper and lower case letters

This library allows for the execution of async ClearBlade messaging functions by providing an MQTT client that will work asynchronously. You can connect to the ClearBlade's Internal Broker for the same system where the code resides, or to an external broker (e.g. ClearBlade Broker in another system on the same or different instance, Mosquitto Broker, etc.).

Reply:
@amoujaes1 I re-worded the thing, do you mind updating it based on that?